### ISSUE TITLE
[BUG] HTTP link href attribute is truncated when embedded in context with special characters

### ISSUE THREAD

**Issue by fracting** [oxen-io/session-ios#770]
### Code of conduct

- [X] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-ios/blob/master/CODE_OF_CONDUCT.md)

### Self-training on how to write a bug report

- [X] I have learned [how to write a bug report](https://www.browserstack.com/guide/how-to-write-a-bug-report)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

an HTTP link

[https://oxen.io/upgrade-jan-2023](https://oxen.io/upgrade-jan-2023
)

is render as

[https://oxen.io/upgrade-jan-20](https://oxen.io/upgrade-jan-20)23

and leads to a 404 error when clicking on that link

### Expected Behavior

`https://oxen.io/upgrade-jan-2023` should link to https://oxen.io/upgrade-jan-2023, the HTTP href attribute should not be truncated, no 404 error should happen

### Steps To Reproduce

1. fresh new account on Session iOS
3. someone sends some content with both special characters and an HTTP link
4. on iOS client, the HTTP href attribute is truncated like [https://oxen.io/upgrade-jan-20](https://oxen.io/upgrade-jan-20)23
5. if the user clicks on that link, the browser opens a broken link https://oxen.io/upgrade-jan-20

Here are the content with special characters which triggers the bug
```
🚨 ATTENTION SERVICE NODE OPERATORS 🚨

Commencing January 26, there will be a mandatory $OXEN Service Node upgrade. Service Node operators will need to complete this upgrade by February 9.

For more information on the changes, and how to update, please see the blog post below.

https://oxen.io/upgrade-jan-2023
```

Desktop seems work fine

### iOS Version

iOS 16.0

### Session Version

2.2.4

### Anything else?

<img width="779" alt="Screen Shot 2023-01-20 at 11 15 46 am" src="https://user-images.githubusercontent.com/194342/213591734-2ea2ab39-cab6-4c34-8f65-fb12b97f2733.png">

**Comment by fracting** [comment]
Update:

This is a reduced test case:

`🚨https://oxen.io`

is rendered as

🚨[🚨https://oxen.i](https://oxen.i)o

the last `o` is presented in text form but not in the HTTP href attribute.

**Comment by RyanRory** [comment]
Hi @fracting, happy to report that this issue should be fixed with our latest release. I'll close this issue for now.

**Comment by fracting** [comment]
Confirming fixed in 2.2.6, thanks!
